en/mkdocs.yml

```diff
@@ -169,9 +169,8 @@ nav:
             - Device-Based: guides/adaptive-auth/device-based-adaptive-auth.md
             - Login Attempts-Based: guides/adaptive-auth/login-attempts-based-adaptive-auth.md
             - ACR-Based: guides/adaptive-auth/acr-based-adaptive-auth.md
-            - Risk-Based Scenarios: 
-              - Elk analytics-Based: guides/elk-analytics/elk-configuring-risk-based-adaptive-authentication.md
-              - TypingDNA-Based: guides/adaptive-auth/typingdna-based-adaptive-auth.md
+            - Elk analytics-Based: guides/elk-analytics/elk-configuring-risk-based-adaptive-authentication.md
```

Comment:
hope this was intentional?
@himeshsiriwardana

Reply:
yeah Nilmini. @divyaamunugama pointed this out. It doesn't make sense to have a section called "Risk-based" since all adaptive authentication scripts are risk-based.